xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxLists.uk.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc>
+  <web>XWiki</web>
+  <name>XWikiSyntaxLists</name>
+  <language>uk</language>
+  <defaultLanguage>en</defaultLanguage>
+  <translation>1</translation>
+  <parent>XWiki.XWikiSyntax</parent>
+  <creator>xwiki:XWiki.Admin</creator>
+  <author>xwiki:XWiki.Admin</author>
+  <customClass/>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <version>1.1</version>
+  <title></title>
+  <defaultTemplate/>
+  <validationScript/>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+<content></content>
```

Comment:
We should probably wait for a translation here or copy the english version to avoid having an empty page for this language.

Reply:
Indeed not great to have an empty translation here because there won't be any fallback I think. Another possibility is deal with this issue directly in the syntax application (fallback on Locale.ROOT when empty since it knows in this specific context it does not make sense).

Reply:
>  Can't we simply remove it from weblate ?

@atallahade tell me if I'm right, but the file is generated by Weblate script according to the given translation, right? Isn't it possible to not generate it if the translation's missing?

Reply:
More precisely can't we check here that the content is not empty: https://github.com/xwiki/xwiki-dev-tools/blob/master/weblate-scripts/pre_commit.py#L47

Reply:
I was thinking more about something like ignoring any translation resource which does not have any translation value set. That would also avoid committing useless key=value based translations resources even if they don't really cause a problem.

Reply:
Yes we can easily check if the translation for a language is empty (i.e. has no key / value set) and not generate / remove the translation file.

Reply:
This one has been removed from the PR.

Reply:
Looks like there is others.

Reply:
I'm not sure if I will be able to do it soon. What we can do is dropping the empty translation commits and, in weblate, copy the English translation in the empty content tag where only the title has been translated (I did that with one so far).

Reply:
Why not just removing the commit?

Reply:
Some documents have the title translated and the content missing (empty). We can keep the translated title and use the English content instead but still "tell" weblate that the content needs to be translated.

Reply:
@surli sounds good too. It's certainly much simpler in many ways and frankly I don't see the point of translating only the title (if we really end up with a use case like this later we can then re-discuss this behavior).